dashboard.go

```diff
@@ -681,7 +681,7 @@ type HeatmapViewProperties struct {
 	Type              string           `json:"type"`
 	Queries           []DashboardQuery `json:"queries"`
 	ViewColors        []string         `json:"colors"`
-	BinSize           int32            `json:"binSize"`
```

Comment:
can we leave this as is, this may have unintended consequences? What's the motivation here to change it?